src/rest/APIRequest.js

```diff
@@ -80,3 +107,8 @@ class APIRequest {
 }
 
 module.exports = APIRequest;
+
+/**
+ * @external HTTPMethod
```

Comment:
Shouldn't this be declared on the website instead, just like other things like Function and string are?

Reply:
I didn't think so because they are not under the javascript scope. However, even if we end up adding it via the website, I would like to keep them here until said PR is merged.

Reply:
Let's hear what @iCrawl thinks then

Reply:
I think this should be fine honestly.